### ISSUE TITLE
Add a "purchase flow" to test if users will click through to buy 

### ISSUE THREAD

**Issue by codelitt** [codelitt/launchpage-rails#10]
The real test for many startups is will people pay for this. We should add a fake purchase flow in which the user would click through to purchase and we just show an error that it is currently not accepting orders but can still track how many people 'intended' to purchase.

**Comment by avinoth** [comment]
@codelitt instead of placing a *fake* purchase button and misleading people, you can very well place a button "Will you pay for this" or more straightforward "Do you Like this?" that's it. You can still get to know if users willing to pay for it.

**Comment by codelitt** [comment]
@avinoth that's a fair point. The intention wouldn't be to mislead, rather, the intention is to see if they will actually start to go down the purchase flow.

**Comment by kaiomagalhaes** [comment]
I agree with @codelitt because the "Will pay for this" is very misleading for the startup, its easy for anyone say "I will do". The informations gathered with the "Buy this product" are much more valuable considering that it is the real intent of the user.
And beyond that, if the user put his email this already means that he like the idea of the product. So add a button " Do you like this" may be redundant. So the "Buy this product"(example) is much more direct.
